apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsProseControl.tsx

```diff
@@ -91,8 +91,11 @@ export function JsonFormsProseControl({
   // Needed to force the value to be set when user clicks on "Go back to editing" in the exit modal
   useEffect(() => {
     if (data !== undefined) {
+      const selection = editor?.state.selection
+      if (!selection) return
       // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-      editor?.commands.setContent(data)
+      editor.commands.setContent(data, false)
```

Comment:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/80de97b6-fdd5-412f-b451-1b869b51dba0" />

not sure if we need this second argument?

Reply:
resolved in https://github.com/opengovsg/isomer/pull/979/commits/cb8ab4f586a060493951334d2614f65cefa5266d